src/plugins/index_pattern_management/public/index.scss

```diff
@@ -0,0 +1,2 @@
+@import './components/field_editor/index';
+@import './components/create_index_pattern_wizard/components/step_time_field/components/time_field/time_field';
```

Comment:
This is way too deep of a nested import. Can you just import the `time_field.scss` file directly in the component TS file instead? https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#syling-with-sass

Reply:
Unfortunately no. I don’t know for sure, but styles do not load.

Reply:
How were you attempting it?

Reply:
@cchaos and @VladLasitsa - I think I got it, you can look at the commit. Its easier than expected - just had to forget about some bad habits.